code/identity/src/main/java/com/adobe/marketing/mobile/identity/IdentityExtension.java

```diff
@@ -864,7 +883,7 @@ boolean handleSyncIdentifiers(
                     LOG_SOURCE,
                     "handleSyncIdentifiers : Ignored the Sync Identifiers call because the privacy"
                             + " status was opt-out.");
-            return true; // cannot process event due to privacy setting, remove from queue
+            return false;
```

Comment:
not sure why we handle privacy again here. we should update L837 to get privacy from latest config ss and remove this repeating check L879->L886

Reply:
It's the same as in Android 1.x.  iOS cleaned this up a bit to only use "latestValidConfig". That will require a bit more changes to the code and test cases, so I'll defer that to another PR.